packages/ses/README.md

```diff
@@ -573,9 +575,10 @@ const compartment = new Compartment();
 compartment.evaluate(program);
 ```
 
-If the host program (Figure 2) arranges for the compartment's `globalThis` to
+If the host program arranges for the compartment's `globalThis` to
```

Comment:
Why remove the "Figure" references by leave the labeling in the figures themselves?

Reply:
Good point. I just added https://github.com/endojs/endo/pull/1613/commits/9a22e01c10d431775fe14bdfdde7aac60aef61ea to replace the figure numbering with linkable section headings; PTAL.